CiscoMeraki/.secrets-ignore

```diff
@@ -0,0 +1 @@
+11.11.1.111
```

Comment:
Why do we need this?

Reply:
In previous PRs, we received your comments to add this file.
/.secret-ignore
  See - Cisco SMA
The need for this is at your discretion.

Reply:
I deleted the irrelevant files.